experimental/packages/exporter-trace-otlp-http/src/platform/node/util.ts

```diff
@@ -96,6 +95,7 @@ export function sendWithHttp<ExportItem, ServiceRequest>(
       break;
     }
     default:
+      req.setHeader('Content-Length', Buffer.byteLength(data));
       req.write(data);
       req.end();
```

Comment:
actually these 3 lines could be combined to one as node adds a content-length header if it knows the size: `req.end(data)`.

Reply:
Ah! Got it. Thanks for the feedback.